channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -106,7 +107,8 @@ export class Sender extends BaseTelemetryPlugin implements IChannelControlsAI {
             instrumentationKey: undefined,
             namePrefix: undefined,
             samplingPercentage: undefined,
-            customHeaders: undefined
+            customHeaders: undefined,
+            convertCustomDimensionUndefinedField: undefined,
```

Comment:
nit: Can we make this a little smaller as this is public and therefore all references will never get minified.

Reply:
Yeah.. agree.. what about "convertUndefined"?